src/comp/comp.py

```diff
@@ -24,48 +24,52 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [h.name for h in humans if h.name[0] == 'D']
+# a = [h.name for h in humans if h.name.startswith('D')]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [h.name for h in humans if h.name[len(h.name) - 1] == 'e']
```

Comment:
```suggestion
b = [h.name for h in humans if h.name[- 1] == 'e']
```
You could also do this